kubeflow/common/iap-secret/Makefile

```diff
@@ -0,0 +1,14 @@
+# The name of the context for your Kubeflow cluster
+# NAME=$(shell yq r ./Kptfile 'openAPI.definitions."io.k8s.cli.setters.name".x-k8s-cli.setter.value')
+# KFCTXT=$(NAME)
+
+.PHONY: apply
+apply: iap-secret
+
+.PHONY: iap-secret
+iap-secret: check-iap
+	kubectl --context=$(KFCTXT) -n istio-system create secret generic kubeflow-oauth --from-literal=client_id=$(CLIENT_ID) --from-literal=client_secret=$(CLIENT_SECRET) --dry-run=client -o yaml | kubectl apply -f -
+
+.PHONY: check-iap
+check-iap:
+	../../hack/check_oauth_secret.sh
```

Comment:
move this script inside?

also, this folder seems to better belong to ingress folder

Reply:
Sounds good, moved shell script to here. Since I want to apply this iap secret in the middle of the loop that applies components. I make this folder and expose the `make apply` target.